genenetwork/genenetwork3
# Heatmap generation

#### Description

Migrates the following sections of code

- Use of `neworder` variable at https://github.com/genenetwork/genenetwork1/blob/master/web/webqtl/heatmap/Heatmap.py#L120 and in the `web.weqtl.heatmap.Heatmap.draw` function
- retrieval of strains with values as at https://github.com/genenetwork/genenetwork1/blob/master/web/webqtl/heatmap/Heatmap.py#L215-221

#### How should this be tested?

Tests were added for these functions as the migration was happening; they can be run the usual way.

The review could also look to verify whether the migration was done correctly.

#### Any background context you want to provide?

https://github.com/genenetwork/gn-gemtext-threads/blob/main/topics/gn1-migration-to-gn2/clustering.gmi

#### What are the relevant pivotal tracker stories?

https://github.com/genenetwork/gn-gemtext-threads/blob/main/topics/gn1-migration-to-gn2/clustering.gmi

#### Screenshots (if appropriate)

N/A

#### Questions

- Do we still use QTLReaper? I seem to remember the intention to move away from it.
- If not, what do we use now?
